lib/pa_ess/http_updater.ex

```diff
@@ -34,11 +28,20 @@ defmodule PaEss.HttpUpdater do
     http_poster = opts[:http_poster] || Application.get_env(:realtime_signs, :http_poster_mod)
     queue_mod = opts[:queue_mod] || MessageQueue
 
+    max_send_rate_per_sec =
+      (32 / Application.get_env(:realtime_signs, :number_of_http_updaters))
+      |> Float.ceil()
+      |> Kernel.trunc()
+
+    avg_ms_between_sends = round(1000 / max_send_rate_per_sec)
+
     GenServer.start_link(
       __MODULE__,
       http_poster: http_poster,
       queue_mod: queue_mod,
-      updater_index: opts[:updater_index]
+      updater_index: opts[:updater_index],
+      max_send_rate_per_sec: max_send_rate_per_sec,
```

Comment:
nit: It looks like this value doesn't get used later on, so we don't technically need to store it in state

Reply:
Ah yeah that's a good point, will remove that then merge